frontend/packages/core/src/card.tsx

```diff
@@ -126,20 +119,90 @@ const BaseCardContent = styled.div<SpacingProps>`
   ${spacing}
 `;
 
-const StyledCardContent = styled(BaseCardContent)({
+const StyledCardContentContainer = styled.div({
   "> .MuiPaper-root": {
     border: "0",
     borderRadius: "0",
   },
+  overflow: "hidden",
 });
 
+const BaseCardActionArea = styled(CardActionArea)<SpacingProps>`
+  ${spacing}
+`;
+
+const StyledCardActionArea = styled(BaseCardActionArea)({
+  ":hover": {
+    backgroundColor: "#F5F6FD",
+  },
+
+  ":active": {
+    backgroundColor: "#D7DAF6",
+  },
+});
+
+const StyledExpandButton = styled(IconButton)({
+  width: "32px",
+  height: "32px",
+  color: "#3548D4",
+});
+
+interface CardContentCollapsibleProps {
+  title: string;
+  dualStateIcon?: { true: React.ReactElement; false: React.ReactElement };
```

Comment:
lmk if your thoughts on this. initially also had the option to just pass a single icon but since the designs are dual-state thought maybe we'd want to be opinionated 
```
{
  icon?: React.ReactElement
  dualStateIcon?: ...
}
```